RadarSDK/remodel/RadarJSONPlugin.ts

```diff
@@ -0,0 +1,766 @@
+export interface TypeMatchers<T> {
```

Comment:
This file is now in a hack-n-dirty state. will clean up if we decide to use the auto code generator